DeepSpeech.py

```diff
@@ -13,6 +13,7 @@
 import progressbar
 import shutil
 import tensorflow as tf
+import tensorflow.compat.v1 as tfv1
```

Comment:
How much work is it to use newer interfaces ? Are they ready yet ? I see one change to `tf.io` for example, at the end.
Should we file a bug to track and handle updating and removing those `tfv1` ?

Reply:
A lot of work :)

Reply:
And they're not ready.